sbtc-core/src/signer/mod.rs

```diff
@@ -0,0 +1,116 @@
+/// An sBTC signer configuration
+pub mod config;
+
+use std::collections::HashMap;
+
+use crate::SBTCError;
+use crate::{signer::config::Config, SBTCResult};
+use bitcoin::{Network, PrivateKey, PublicKey, Transaction as BitcoinTransaction};
+use p256k1::ecdsa;
+use url::Url;
+use wsts::{bip340::SchnorrProof, common::Signature};
+
+#[derive(Default, Clone, Debug)]
+/// Signers' data required for weighted distributed signing
+pub struct Signers {
```

Comment:
I don't like the name `Signers` here.  The encapsulated data is the public keys for all signers, indexed by their signer IDs and key IDs, not the signers themselves.

I think we called this `SignerKeys` or `SignerPublicKeys` previously?  Since we're in the `signers` module here I'd probably just call it `PublicKeys`, and skip the excess `Signer` token.